example/cumulus-tf/thin-egress-app/bucket_map.yaml.tmpl

```diff
@@ -14,6 +14,6 @@ MAP:
 %{ if length(public_buckets) > 0 ~}
 PUBLIC_BUCKETS:
 %{ for bucket in public_buckets ~}
-  - ${bucket}
+  ${bucket}: "${bucket} is a public bucket"
```

Comment:
Apparently the way our bucket map for PUBLIC_BUCKETS was defined was wrong: https://github.com/asfadmin/thin-egress-app/#public-buckets

I confirmed that the public bucket access works properly with these changes